Receiver/process_input.py

```diff
@@ -0,0 +1,13 @@
+def parameters_statistics(temperature,charge_rate):
+      param_dict = {}
+      param_dict["temperature"] = calcParam(temperature)
+      param_dict["charge_rate"] = calcParam(charge_rate)
+      return param_dict
+        
+def calcParam(battery_parameter):
```

Comment:
Use a more specific name - what would you like readers to understand, while seeing this name? `calculateStatistics`?